src/lib.rs

```diff
@@ -279,9 +295,14 @@ impl<S: Subscriber + for<'a> LookupSpan<'a>> Layer<S> for WASMLayer {
                     .file()
                     .and_then(|file| meta.line().map(|ln| format!("{}:{}", file, ln)))
                     .unwrap_or_default();
+                let timestamp = if self.config.log_time {
+                    chrono::Local::now().format("%a %d %T%.f").to_string()
+                } else {
+                    String::new()
+                };
                 if self.config.use_console_color {
                     log4(
-                        &format!("%c{}%c {}%c{}", level, origin, recorder),
+                        &format!("{} %c{}%c {}%c{}", timestamp, level, origin, recorder),
```

Comment:
This appears that if the timestamp is disabled, that we'll end up accidentally adding a space before all logs.

Perhaps we can avoid that by simply adding a space at the end of `"%a %d %T%.f "`. I'm not sure if that's how the formatter works for chrono. Alternatively, just adding a space to the end of the `timestamp` var.

Reply:
oops